src/common/mfu_daos.c

```diff
@@ -218,10 +218,32 @@ static int daos_check_args(
         return 0;
     }
     
+    /* if passed in values for src/dst are both uuids ignore the case when comparing */
+
     /* Determine whether the source and destination
      * use the same pool and container */
-    bool same_pool = (strcmp(da->src_pool, da->dst_pool) == 0);
-    bool same_cont = same_pool && (strcmp(da->src_cont, da->dst_cont) == 0);
+    int rc1, rc2 = 0;
```

Comment:
It's a bit strange to initialize rc2 but not rc1 here.  Maybe initialize both or neither?  I'm fine either way.

Reply:
@adammoody oh yeah I just changed it to neither.